File: gaia-sdk-java/gaia-sdk-graphql/src/main/kotlin/gaia/sdk/request/input/CreateIntentImpulse.kt
package gaia.sdk.request.input

import gaia.sdk.client.Type
import gaia.sdk.request.intf.*
import gaia.sdk.client.Input
import gaia.sdk.Uuid
import gaia.sdk.Timestamp
import gaia.sdk.Long
import gaia.sdk.Struct
import gaia.sdk.request.input.*
import gaia.sdk.request.enumeration.*

/**
 * The specification to create an intent instance
 */
class CreateIntentImpulse(val identityId:Uuid, 
   val qualifier:String, 
   val appendent:String, 
   val utterance:Struct, 
   val labellist:Struct, 
   val version:String) : Input()

File: gaia-sdk-python/graphql/request/input/CreateIntentImpulse.py


from typing import Callable
from api.VariableRegistry import VariableRegistry


from dataclasses import dataclass


@dataclass
class CreateIntentImpulse():
    """
    The specification to create an intent instance
    """

    identityId: str
    qualifier: str
    appendent: str
    utterance: dict
    labellist: dict
    version: str
